src/main/resources/application.yml

```diff
@@ -1,5 +1,13 @@
 spring:
   application.name: gateway
+  cloud:
+    gateway:
+      discovery:
+        locator:
+          enabled: true
+          url-expression: "'http://'+serviceId"
```

Comment:
If I understand the service discovery in `spring-cloud-gateway` and `spring-cloud-kubernetes-discovery` correctly, this config would create routes for every K8s service deployed, which sounds risky.

Can we use an [`include-expression`](https://github.com/spring-cloud/spring-cloud-gateway/issues/185) here to target specific services, perhaps with some annotation like `alfresco.com/expose:true` (similar to `fabric8.io/expose`)?